syntaxes/Major.tmLanguage

```diff
@@ -242,15 +242,6 @@
 				</dict>
 			</array>
 		</dict>
-		<key>catch-statement-keyword</key>
-		<dict>
-			<key>comment</key>
-			<string>catch-statement</string>
-			<key>match</key>
-			<string>\b(catch|do)\b</string>
-			<key>name</key>
-			<string>kewyord.control.catch.motoko</string>
```

Comment:
"kewyord" !?

Reply:
Yep. This was all from the original syntax highlighting code, which is absolutely chock-full of subtle mistakes like this. I've considered rewriting this entire file several times, but it's probably not worth it due to the possibility of introducing other unexpected issues.